static/app/views/alerts/rules/teamFilter.tsx

```diff
@@ -115,8 +115,9 @@ const InputWrapper = styled('div')`
 
 const StyledInput = styled(Input)`
   border: none;
-  border-bottom: 1px solid transparent;
   border-radius: 0;
+  border-bottom: solid 1px ${p => p.theme.border};
```

Comment:
This is a bit different looking than anything else, you sure you want to introduce this right now?

Reply:
This whole selector is out of spec. We should probably switch it to `DropdownAutocomplete` or `CompactSelect` at some time in the future. For now, a bottom border just brings it more into line w/ `DropdownAutocomplete`.